package-lock.json

```diff
@@ -0,0 +1,561 @@
+{
```

Comment:
We don't want to commit package-lock I think. This should be deleted

Reply:
The package-lock currently isn't used by the template at build-time, so having it committed does nothing for us effectively.

I'm okay with removing it here but I hope that in the future the template can take it into account and that we can use it, because at least here specifically it's pretty useful (for these SDKs the required versions are quite particular and it can just break after an install).

Will delete it here (as it's still in the commit history and we can bring it back if we ever encounter issues).